all-repos.yaml

```diff
@@ -205,3 +205,4 @@
 - https://github.com/scop/pre-commit-shfmt
 - https://github.com/BlankSpruce/gersemi
 - https://github.com/realm/SwiftLint
+- https://gitlab.com/bmares/pre-commit-check-json5
```

Comment:
nice!  would it be possible to rename it to just `check-json5` -- that would better differentiate it from being a "pre-commit" affiliated project vs just a project that can be integrated with pre-commit

Reply:
additionally it would be best to avoid using the `pre_commit_hooks` namespace

Reply:
Ah, ok... just a few minutes, I'll have to update the repos and the README. Thanks for the quick response! ...

Reply:
@asottile, done! And thanks in general for your awesome work. I'm a big fan. :smile: